src/cli/adapter/prettier.ts

```diff
@@ -172,7 +171,7 @@ export const runPrettier = async (
   if (result.errored.length) {
     logger.plain(`Flagged ${logger.pluralise(result.errored.length, 'file')}:`);
     for (const { err, filepath } of result.errored) {
-      logger.warn(filepath, ...(err ? [inspect(err)] : []));
+      logger.warn(filepath, ...(err ? [String(err)] : []));
```

Comment:
We get good-enough error messages without a full-blown `inspect`, which enumerates additional properties including a verbose stack trace.